lib/mongo/auth.rb

```diff
@@ -58,6 +58,7 @@ module Auth
     #
     # @since 2.0.0
     SOURCES = {
+      aws: Aws,
```

Comment:
Do you think it would make sense to eventually change the other auth class names to all be written in the same style (i.e. `Ldap`, `Scram`, etc)?

Reply:
I'd like to but we can't do this until driver 3.0.